packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1183,6 +1336,153 @@ export class Combobox
     });
   }
 
+  renderAllSelectedIndicatorChip(): VNode {
+    const {
+      compactDisplayMode,
+      isAllSelected,
+      scale,
+      selectedVisibleChipsCount,
+      setAllSelectedIndicatorChipEl,
+    } = this;
+    const label = this.messages.allSelected || "All selected";
+    return (
+      <calcite-chip
+        class={{
+          chip: true,
+          [CSS.chipInvisible]: !(
+            isAllSelected() &&
+            !selectedVisibleChipsCount &&
+            !compactDisplayMode
+          ),
+        }}
+        scale={scale}
+        title={label}
+        value=""
+        ref={setAllSelectedIndicatorChipEl}
+      >
+        {label}
+      </calcite-chip>
+    );
+  }
+
+  renderAllSelectedIndicatorChipCompact(): VNode {
+    const { compactDisplayMode, isAllSelected, scale, selectedVisibleChipsCount } = this;
+    const label = this.messages.all || "All";
+    return (
+      <calcite-chip
+        class={{
+          chip: true,
+          [CSS.chipInvisible]: !(
+            isAllSelected() &&
+            !selectedVisibleChipsCount &&
+            compactDisplayMode
+          ),
+        }}
+        scale={scale}
+        title={label}
+        value=""
+      >
+        {label}
+      </calcite-chip>
+    );
+  }
+
+  renderSelectedIndicatorChip(): VNode {
+    const {
+      compactDisplayMode,
+      displayMode,
+      getSelectedItems,
+      isAllSelected,
+      scale,
+      selectedHiddenChipsCount,
+      selectedVisibleChipsCount,
+      setSelectedIndicatorChipEl,
+    } = this;
+    let chipInvisible, label;
+    if (compactDisplayMode) {
+      chipInvisible = true;
+    } else {
+      if (displayMode === "single") {
+        const selectedItemsCount = getSelectedItems().length;
+        if (isAllSelected()) {
```

Comment:
Either call this with `this` (e.g., `this.isAllSelected()`) or store the value and reuse. The way it's called is requires you to unnecessarily bind the function where it's declared. Applies to other assigned, bound functions.